src/index.md

```diff
@@ -28,4 +28,9 @@ This flexibility comes from how we use key brand elements – the wider colour p
 
 These guidelines show how to apply the brand in different contexts to meet the needs of people using GOV.UK every day.
 
-![TODO](./inform-inspire.svg)
+{% informInspire %}
+
+  <div class="app-inform-inspire__bar">Web</div>
```

Comment:
I think this is okay. But keep in mind that these are likely to need to change again, so might need to be made headings if more content explaining the channels will follow after each channel name.
We'll get there in a different PR when the homepage changes have been decided.